src/transaction.py

```diff
@@ -744,6 +764,10 @@ class StakingInterestReportEntry(InterestReportEntry):
     event_type = "Staking Einkünfte"
 
 
+class MarginReportEntry(InterestReportEntry):
+    event_type = "Margin-Trading"
```

Comment:
Does the labels from `InterestReportEntry` make sense here? All your MarginGain/MarginLoss.change are positive. So that the report should state that you always receive ("Erhalten am") coins. I believe that you have to update the labels. Also "Wert in EUR" doesn't make sense as we do not evaluate the value of the "bought" coins.

Reply:
Changed to `TaxReportEntry`.
"Wert in EUR" makes sense for gains/losses/fees received/paid in crypto coins. However, since it's the same as "Gewinn/Verlust in EUR", I removed the column.
Not sure about the date (as it could be gain or loss), what do you think about "Erhalten oder ausgegeben am"?